scripts/compile-jsonschema.sh

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
```

Comment:
Add a description of what the script does underneath the shebang with usage. Example:

```sh
#
# Deletes old Google App Engine deployments of Terra UI in an environment. You
# MUST have jq installed to be able to use this script.
#
# USAGE: ./delete-old-app-engine-versions.sh ENV
#   ENV must be one of dev, alpha, or staging
#
```